docs/adr/0016-pr-review.md

```diff
@@ -0,0 +1,64 @@
+# Reviewing Pull Requests
+
+* Status: accepted
+* Deciders: tumido, HumairAK, lars, goern, 4n4nd
+* Date: 2021-05-20
+
+Technical Story: [#93](https://github.com/operate-first/blueprint/issues/93)
+
+## Context and Problem Statement
+
+Each community has its own `contributing.md` file which outlines how are users expected to contribute to the community. That guide for contributions should also clearly state what will happen once a PR is submitted and how reviews are performed.
+
+
+## Considered Options
+
+* Using inherited processes from AI-CoE - Prow
+* Building upon Prow and modifying and evolving them to operations needs
+
+## Decision Outcome
+
+Chosen option: "Building upon Prow and modifying and evolving them to operations needs"
+
+### Prerequisistes
+
+We use Prow to automate review process. That means we use following setup:
+
+1. Each repository contains an `OWNERS` file:
+   1. There is always one `OWNERS` file in a repository root folder
+   2. If needed additional `OWNERS` files can be created within nested folders. This file takes precedence for all resources within this folder.
+2. `OWNERS` file contains several entries:
+   1. `approvers` is a list of GitHub users that are allowed to use `/approve` chat-ops command. Role of approvers is to gate PRs - they verify that changes in PRs respect our architectural decisions and are in line with the heading of the project.
+   2. `reviewers` is a list of GitHub users that are allowed to use `/lgtm` chat-ops command. Their role is to ensure changes in PRs are implemented properly and meet our coding standards.
+
+### Process
```

Comment:
The process looks like we can just copy paste it into the contributing.md files, is this what you had in mind?

Reply:
Yes, I think we can either link the ADR or copy it over there.